dask_expr/_categorical.py

```diff
@@ -127,3 +134,31 @@ class AsUnknown(Elemwise):
     @functools.cached_property
     def _meta(self):
         return clear_known_categories(self.frame._meta)
+
+
+class Categorize(Blockwise):
+    _parameters = ["frame", "categories", "index"]
+    operation = staticmethod(_categorize_block)
+    _projection_passthrough = True
+
+    @functools.cached_property
+    def _meta(self):
+        meta = _categorize_block(
+            self.frame._meta, self.operand("categories"), self.operand("index")
+        )
+        return meta
+
+
+class GetCategories(ApplyConcatApply):
```

Comment:
This should implement simpliy_down, so that we inject a Projection before the ``GetCategories`` so that we can drop unnecessary columns

Reply:
You can also do this before calling ``GetCategories``, that might make things a little bit easier

Reply:
Done.